docs/manual/troubleshooting.tex

```diff
@@ -947,18 +947,32 @@ \subsection{Configure Eclipse to edit the Checker Framework\label{building-eclip
 \begin{enumerate}
 \item Follow the instructions earlier in Section~\ref{build-source} to
 clone and build all projects from their sources.
+\item Follow the instructions in the \<README-eclipse> document found in the
+\<\$CHECKERFRAMEWORK/../annotation-tools/annotation-file-utilities/lib>
+directory to download JDK 8 source code and create symbolic links to
+\<tools.jar> and \<src.zip>.
 \item Download Eclipse from
 the \href{https://www.eclipse.org/downloads/}{official Eclipse website}
 and install it.
-\item Run Eclipse.
-\item Enter the main Eclipse working screen and in the ``File'' menu,
-select ``Import'' $\rightarrow$ ``General'' $\rightarrow$ ``Existing
-Projects into Workspace''.
+\item Run Eclipse and enter the main Eclipse working screen.
+\item Checker Framework's sources depend on classes packaged in \<rt.jar> from
```

Comment:
Everything depends on `rt.jar`, so this sentence reads strange. Do you mean `tools.jar`?

Reply:
Agreed that it reads strange. Will fix that part.

As to dependencies, do we want to have sources for classes that are in `rt.jar` or only for classes in `tools.jar`?